lang/es.json

```diff
@@ -229,6 +230,7 @@
 	"label.credit_card_info": "Información bancaria",
 	"label.crypto_made_easy": "Cripto de manera fácil",
 	"label.curate_projects": "Cura Proyectos",
+	"label.currently_this_feature_is_only_available": "Currently this feature is only available on EVM chains.",
```

Comment:
This in spanish would be

"Actualmente esta funcionalidad solo esta disponible en cadenas EVM"

Reply:
also this one is fixed here https://github.com/Giveth/giveth-dapps-v2/pull/3689